src/ansible_compat/constants.py

```diff
@@ -0,0 +1,31 @@
+"""Constants used by ansible_compat."""
+import os.path
+import sys
+
+# mypy/pylint idiom for py36-py38 compatibility
+# https://github.com/python/typeshed/issues/3500#issuecomment-560958608
+if sys.version_info >= (3, 8):
+    from typing import Literal  # pylint: disable=no-name-in-module
+else:
+    from typing_extensions import Literal
+
+# Minimal version of Ansible we support for runtime
+ANSIBLE_MIN_VERSION = "2.9"
+
+# Based on https://docs.ansible.com/ansible/latest/reference_appendices/config.html
+ANSIBLE_DEFAULT_ROLES_PATH = (
+    "~/.ansible/roles:/usr/share/ansible/roles:/etc/ansible/roles"
```

Comment:
Some software doesn't resolve `~` but resolves `$HOME`. Should we care about it in this string?

Reply:
Is ok, we resolved it ourselves.